services/frontend/src/scenes/ProjectDetails/ProjectDetailsLayout.js

```diff
@@ -38,12 +38,12 @@ const sections = [
     }
 ];
 
-const renderSectionComponent = (sectionName, sectionProps) => {
+const renderSectionComponent = (sectionName, sectionProps, contributors) => {
```

Comment:
Umm, I don't think the extra `contributors` parameter is actually _doing_ anything. `renderSectionComponent` never gets called with a third argument, so the `contributors` parameter is never actually filled with a value. The `Contributors` component only works because `contributors` gets implicitly passed through `sectionProps`.

As such, you can just remove the `contributors` parameter here.